src/components/ThemeProvider/utils/__tests__/__snapshots__/themeDefaultStyle.test.ts.snap

```diff
@@ -6,6 +6,8 @@ exports[`Theme Default Style generate darkTheme default style 1`] = `
   "fontFamily": "Inter",
   "fontSize": "14px",
   "fontWeight": "400",
+  "height": "100%",
```

Comment:
?

Reply:
It is useful for mantaining the correct layout when adding ThemeProvider, otherwise the content is stretched to its minimum size